packages/react-server/src/logger/logger.ts

```diff
@@ -1,50 +1,87 @@
-import {Context} from 'koa';
+/* eslint-disable no-process-env */
+import {Context, Request} from 'koa';
 import chalk from 'chalk';
 import {KoaNextFunction} from '../types';
 
 export const LOGGER = Symbol('logger');
-const PREFIX = chalk`{underline sidecar} ⁓ `;
+const PREFIX = chalk`{underline [React Server]}  `;
```

Comment:
IMO, `[React server]` is better suited in logs for this than `sidecar ~`. I can change it back if we don't like it